src/main.jl

```diff
@@ -10,9 +10,16 @@ function sim!(P, C, dt)
 end
 
 function sim!(P, C; dt = 0.1ms, duration = 10ms)
-    for t = 0ms:dt:(duration - dt)
-        sim!(P, C, dt)
-    end
+	sized = duration/dt
```

Comment:
Please use 4-space indentation, not tabs.

Reply:
fixed